resources/frameworks.yaml

```diff
@@ -94,3 +94,11 @@ GAMA:
 autoxgboost:
   version: '0.0.0.9000'
   project: https://github.com/ja-thomas/autoxgboost
+
+supervised:
```

Comment:
didn't notice this before, but can you give a more explicit name to your framework (same concern for the directory), I'd recommand `mljarsupervised`.
`mljar_supervised` should also work, but I'm only 95% sure (there may be some name resolution/regexp issues when used with a dataset that also has an underscore in its name).

Reply:
I liked `supervised` :) I will change to `mljarsupervised`.